osmaxx-py/config/settings/common.py

```diff
@@ -375,6 +376,9 @@
     'download_file_name': '%(date)s-%(excerpt_name)s-%(id)s.%(content_type)s.%(file_extension)s',
     'download_chunk_size': 8192,
     'orders_history_number_of_items': 100,
+    'EXTRACTION_PROCESSING_TIMEOUT_TIMEDELTA': timezone.timedelta(
+        hours=env.str('DJANGO_OSMAXX_EXTRACTION_PROCESSING_TIMEOUT_HOURS', default=12)
```

Comment:
I think `env.int` (instead of `env.str`) is what is intended here, because otherwise the conversion from a environment variable isn't handled correctly.